### PR TITLE
perf(metrics) - read from new metrics_counters table

### PR DESCRIPTION
**Pull request by onewland**
This is blocked on #2540 as well as Filippo running a manual backfill

### CODE REVIEW

**Review by fpacifici — APPROVED** [review]

**Comment by fpacifici** [comment]
Please fix the tests

**Comment by onewland** [comment]
I think they will work after #2544 is merged

**Comment by onewland** [comment]
The sentry test issues have to do with the timestamp condition in my materialized view migration. Because of the way sentry snuba tests manipulate timestamps to be in the past, I ran `make test-snuba` locally with an MV without that clause and tests succeeded. 

After tomorrow this shouldn’t be a problem again

File: snuba/migrations/snuba_migrations/metrics/0027_fix_migration_0026.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                source_table_name=self.raw_table_name,
+                table_name=self.table_name,
+                aggregation_col_schema=[
+                    Column("value", AggregateFunction("sum", [Float(64)])),
+                ],
+                mv_name=get_polymorphic_mv_variant_name("counters", self.mv_version),
+                aggregation_states="sumState(count_value) as value",
+                metric_type="counter",
+                target_mat_version=4,
+                appended_where_clause="AND timestamp > toDateTime('2022-03-24 16:00:00')",
```
Comment (fpacifici) [inline snuba/migrations/snuba_migrations/metrics/0027_fix_migration_0026.py:35]: I have already done the migraiton, I think you can just drop this condition.

**Review by fpacifici — APPROVED** [review]

**Pull request merged**